web/pom.xml

```
@@ -616,88 +616,86 @@
-	                    <!--
-	                         A descriptive name that will be used to be able to identify the validation that does not match up (will be displayed in the error message).
-	                    -->
-	                    <name>validating project version</name>
-	                    <!--
-	                         the value that needs the validation
-	                         *Note* : In order to be able to validate the generated git-properties inside the pom itself you may need to set the configuration `<injectAllReactorProjects>true</injectAllReactorProjects>`.
-	                    -->
-	                    <value>${project.version}</value>
-	                    <!--
-	                        the expected value
-	                    -->
-	                    <shouldMatchTo><![CDATA[^.*(?<!-SNAPSHOT)$]]></shouldMatchTo>
-	                </validationProperty>
-	                <!-- the next validationProperty you would like to validate -->
-	            </validationProperties>
-	            <validationShouldFailIfNoMatch>false</validationShouldFailIfNoMatch>
-	        </configuration>
+        <groupId>io.github.git-commit-id</groupId>
+        <artifactId>git-commit-id-maven-plugin</artifactId>
+        <version>4.9.9</version>
+        <executions>
+          <execution>
+            <phase>initialize</phase>
+            <id>get-the-git-infos</id>
+            <goals>
+                <goal>revision</goal>
+            </goals>
+          </execution>
+        </executions>
 
+        <configuration>
+          <dotGitDirectory>${project.basedir}/../.git</dotGitDirectory>
+          <prefix>git</prefix>
+          <dateFormat>yyyy-MM-dd'T'HH:mm:ssZ</dateFormat>
+          <dateFormatTimeZone>${user.timezone}</dateFormatTimeZone>
+          <verbose>false</verbose>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
+          <generateGitPropertiesFilename>src/main/webResources/WEB-INF/classes/git.properties</generateGitPropertiesFilename>
+          <!-- Denotes the format to save properties in. Valid options are "properties" (default) and "json". Properties will be saved to the generateGitPropertiesFilename if generateGitPropertiesFile is set to `true`. -->
+          <format>properties</format>
+          <skipPoms>true</skipPoms>
+          <injectAllReactorProjects>true</injectAllReactorProjects>
+          <failOnNoGitDirectory>false</failOnNoGitDirectory>
+          <failOnUnableToExtractRepoInfo>false</failOnUnableToExtractRepoInfo>
+          <skip>false</skip>
+          <runOnlyOnce>false</runOnlyOnce>
+          <excludeProperties>
+          </excludeProperties>
+          <includeOnlyProperties>
+            <includeOnlyProperty>^git.tags$</includeOnlyProperty>
+            <includeOnlyProperty>^git.branch$</includeOnlyProperty>
+            <includeOnlyProperty>^git.commit.id$</includeOnlyProperty>
+            <includeOnlyProperty>^git.commit.id.describe.short$</includeOnlyProperty>
```

This change seems to fix the missing `git.commit.id.describe-short` property, although as side effect it also includes `git.commit.id.describe`:
```
#Generated by Git-Commit-Id-Plugin
git.commit.id.describe=4.2.3-4-gd294f0a-dirty
git.commit.id.describe-short=4.2.3-4-dirty
```
```suggestion
            <includeOnlyProperty>^git.commit.id.describe.*$</includeOnlyProperty>
```